.github/tests/double_whitspace_test.py

```diff
@@ -7,13 +7,28 @@ def find_bad_lines(file_content):
     """
     internal_bad_lines = []
     for i, line in enumerate(file_content, start=1):
-        if ("  ." or ".  " or "  ," or ",  ") in line:
+        false_patterns = ["  .", ".  ", "  ,", ",  "]
+        if any(x in line for x in false_patterns):
             # Ignore CIG's special char test line
             if not line.startswith("test_special_chars"):
                 internal_bad_lines.append(i)
 
     return internal_bad_lines
 
+def extract_keys_from_lines(file_content, line_numbers):
+    keys = []
+    # Iterate through the given line numbers
+    for line_number in line_numbers:
+         # Ensure the line number is within range
+         if 0 <= line_number < len(file_content):
+             line = file_content[line_number].strip()
```

Comment:
Bei mir kommen die Zeile um eins verschoben raus:
```suggestion
             line = file_content[line_number-1].strip()
```
Hat das bei mir gefixt, denk auch an das border checken oben, wenn du das anpasst ;-)